### DISCUSSION TITLE
Add to the app the option to remove artists thats keeps on the library instead of full delete of all items

### DISCUSSION THREAD

**Discussion by ArreglarYTM** [apastel/ytmusic-deleter#77] [Ideas]
I have a problem related to one artist that had a song previously, but currently no, it seems to be related a bug into YTM, also I make like to the song, but doesn´t dessapear from my artists library. ¿Could you implement that feature on the app?, ¿Google API support this option?
Thank you in advance.
Cheers.

![Captura de pantalla 2024-10-16 195933](https://github.com/user-attachments/assets/f880e235-10d7-4c39-bd4f-65a4c19f00d5)
![Captura de pantalla 2024-10-16 195948](https://github.com/user-attachments/assets/c3f4304a-8b54-440f-bb62-5f191fb88393)

**Comment by apastel** [comment]
Addressed in #78
